Userland/Services/ConfigServer/ConnectionFromClient.cpp

```diff
@@ -31,20 +34,38 @@ static void for_each_monitoring_connection(String const& domain, ConnectionFromC
     }
 }
 
+static void notify_user_of_read_only_file_system()
+{
+    if (!s_user_informed_of_read_only_file_system) {
+        s_user_informed_of_read_only_file_system = true;
+        auto notification = GUI::Notification::construct();
```

Comment:
ConfigServer relying on the GUI feels wrong. What if I'm not running WindowServer?

Reply:
but I want a nice user message... some more smarts required

Reply:
I feel like a simple `dbgln` (maybe with some color applied to it) would work quite OK for this. Trying to render an actual notification through the GUI definitely is overkill.